### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by mishakoshilka**

### CODE REVIEW

File: src/herbivoresAndCarnivores.js [hunk 1]
```diff
@@ -1,15 +1,42 @@
...
+    this.health = health;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
-  // write your code here
+  constructor() {
```
Comment (anastasiiavorobiova) [inline src/herbivoresAndCarnivores.js:14]:
```suggestion
  constructor(name, health) {
```

File: src/herbivoresAndCarnivores.js [hunk 2]
```diff
@@ -1,15 +1,42 @@
...
+    super();
```
Comment (anastasiiavorobiova) [inline src/herbivoresAndCarnivores.js:15]:
```suggestion
    super(name, health);
```

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Almost done!

**Review by Esceype — APPROVED** [review]
Well done🚀🚀🚀

**Pull request closed without merging**